public/regdesk.html

```diff
@@ -4,10 +4,28 @@
 <head>
   <meta charset="UTF-8">
   <title>FC RegDesk</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="color-scheme" content="light dark">
+  <meta name="theme-color" content="#111111" media="(prefers-color-scheme: light)">
+  <meta name="theme-color" content="#eeeeee" media="(prefers-color-scheme: dark)">
+  <link rel="stylesheet" href="regdesk.css" />
 </head>
 
 <body>
-    <h1>This will be an awesome page with stuff!</h1>
+  <div class="container-fluid">
+      <div class="row" style="padding-top: 16px;">
+          <div class="col-3">
+              <div class="d-grid gap-2 col-8 mx-auto mb-3">
+                  <button class="btn btn-primary" id="addprinter">➕ Add Printer</button>
```

Comment:
Oh man I'm torn. Usually you want to avoid emoji characters because they will render differently on different devices (and sony is particularly ugly). Usually you would use something like icons instead [1].

[1] https://fonts.google.com/icons?selected=Material+Icons+Outlined 

But we are guaranteed to always be on chrome, so its not that big of a deal. (And again, temporary code, so not gunna block by any means).

Reply:
In the tradeoff of emoji variance vs adding more dependencies to the project I think we'll be okay with emoji. We aren't really using it as part of a coherent theme so much as using them for their ideographic meaning. That should remain consistent through time (even if the style changes drastically) so I'm generally okay with emoji in my functional systems.

And as you pointed out we're targeting a single browser, so it's less a concern here. WebUSB is only functional in Chromium based browsers anyhow.